Bookish/Bookish.DataAccess/SqlReference.cs

```diff
@@ -12,16 +12,22 @@ namespace Bookish.DataAccess
 {
     public class SqlReference
     {
+        private static IDbConnection db = new SqlConnection("Server = localhost; Database = Bookish; Integrated Security = True; MultipleActiveResultSets = true;");
```

Comment:
It's better to create a connection each time we're going to run a query, because otherwise the app is remaining constantly connected to the DB which has several issues - for example, if the connection goes down we won't automatically r-open it.

But it would be nice to have a static *method* here that can be called by the other methods that need it instead of writing out the connection manually in each one.